README.md

```diff
@@ -1 +1,187 @@
 # Work In Progress - Onfido Java Library
+
+The official Java library for integrating with the Onfido API.
+
+Documentation can be found at <https://documentation.onfido.com>
+
+This library is only for use on the backend, as it uses Onfido API tokens which must be kept secret. If you do need to collect applicant data in the frontend of your application, we recommend that you use one of [the Onfido SDKs](https://developers.onfido.com/sdks/).
+
+## Installation
+
+_Coming soon_
+
+## Getting Started
+
+Import the `Onfido` object, this is the main object used for interfacing with the API:
+
+```java
+import com.onfido.Onfido;
+```
+
+Instantiate and configure an `Onfido` instance with your API token, and region if necessary:
+
+```java
+Onfido onfido = Onfido.builder()
+                .apiToken(<YOUR_API_TOKEN_HERE>)
+                .regionUS() // Defaults to EU region, add this to switch to US
+                .build();
+```
+
+## Making a call to the API
+
+Using your configured and instantiated instance of the `Onfido` object you can make calls to the API by calling one of the methods on the resources inside of it. For example:
```

Comment:
All of this content, the most important part of the README I think, is very well written. Nice one. 

Minor: perhaps make it clearer that this example is for creating an applicant i.e.:

> For example, to create an applicant:

I think it helps the rest of the content flow a bit better.